src/lib/falcor/FalcorRouter.js

```diff
@@ -1,53 +1,55 @@
 import BaseRouter from "falcor-router"
 import { ghostArticleQuery } from 'lib/ghostAPI'
-import { sqlArticleQuery, sqlAuthorQuery } from 'lib/sql'
+import { dbAuthorQuery, dbArticleQuery, dbAuthorArticleQuery, dbInfoPagesQuery, dbArticleIssueQuery,
+dbArticleAuthorQuery, dbLatestIssueQuery, dbCategoryQuery, dbCategoryArticleQuery,
+dbFeaturedArticleQuery, dbEditorPickQuery, dbIssueCategoryQuery, dbIssueQuery } from 'lib/mariaDB'
```

Comment:
I would call this lib/db. Because actually anything SQL compliant will work here.